fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadManager.java

```diff
@@ -410,36 +411,43 @@ public long getMinTaskBeId(String clusterName) throws LoadException {
     // check if the specified BE is available for running task
     // return true if it is available. return false if otherwise.
     // throw exception if unrecoverable errors happen.
-    public long getAvailableBeForTask(long previoudBeId, String clusterName) throws LoadException {
+    public long getAvailableBeForTask(long previousBeId, String clusterName) throws LoadException {
         List<Long> beIdsInCluster = Catalog.getCurrentSystemInfo().getClusterBackendIds(clusterName, true);
         if (beIdsInCluster == null) {
             throw new LoadException("The " + clusterName + " has been deleted");
         }
 
-        if (previoudBeId != -1L && !beIdsInCluster.contains(previoudBeId)) {
+        if (previousBeId != -1L && !beIdsInCluster.contains(previousBeId)) {
```

Comment:
If previous BE is down, this method still return -1 and task will be failed to allocated.
So nothing change?

Reply:
Fixed.